docs/ko/docs/advanced/additional-status-codes.md

```diff
@@ -0,0 +1,41 @@
+# 추가 상태 코드
+
+기본적으로 **FastAPI**는 *경로 연산*에서 반환한 콘텐츠를 해당 `JSONResponse`에 넣어 `JSONResponse`를 사용하여 응답을 반환합니다.
+
+기본 상태 코드 또는 *경로 연산*에서 설정한 상태 코드를 사용합니다.
```

Comment:
Use `경로 작업` or `경로 작동` instead of `경로 연산` Ref: https://github.com/fastapi/fastapi/discussions/3167#discussioncomment-721448

Also, if possible, make it a more natural sentence. I think "`JSONResponse`에 넣어 `JSONResponse`를 사용하여" is unnatural.